recipes/gentle/meta.yaml

```diff
@@ -0,0 +1,40 @@
+package:
+  name: gentle
+  version: 1.9.5
+
+source:
+  git_url: https://github.com/GENtle-persons/gentle-m
+  git_depth: 1
+
+build:
+  run_exports:
+    - {{ pin_subpackage('gentle', max_pin="x") }}
+
+requirements:
+  build:
+    - git
+    - autoconf
+    - automake
+    - make
+    - {{ compiler('cxx') }}
+    - wxwidgets
```

Comment:
wxwidgets and all others below needs to go into a `host` section

Reply:
Done.